homeassistant/components/binary_sensor/knx.py

```diff
@@ -107,7 +107,6 @@ def __init__(self, hass, device):
         """Initialize of KNX binary sensor."""
         self.device = device
         self.hass = hass
```

Comment:
This can be removed. `hass` will be set on the entity when it has been added to home assistant.